doc/develop/extending.rst

```diff
@@ -330,6 +330,46 @@ To read an acquisition file you can use code such as:
     print das[0].metadata
 
 
+Starting odemis from the terminal/console
+=================================================
+
+After setting up the development environment it is possible to start odemis via the terminal.
+It is also possible to specify a specific configuration (*.yaml) file used for staring odemis.
+
+
+Starting odemis
+-----------------------
+
+Odemis can be started from the terminal by typing the following command in the terminal
+(from ``~/development/odemis``)::
+
+    odemis-start
+
+The default configuration file (*.yaml) is used, which can be found and changed in
+``~/development/odemis/install/linux/etc/odemis.conf``.
+
+Starting odemis with configuration file
+----------------------------------------
+
+Odemis can be started using different hardware configuration files (*.yaml).
+There are various examples, hardware tests and simulators available in
+``~/development/odemis/install/linux/usr/share/odemis/``.
+
+Launch odemis with a configuration file by typing the following command in the terminal
+(from ``~/development/odemis``)::
+
+    odemis-start install/linux/usr/share/odemis/sim/sparc2-sim.odm.yaml
+
+
+Starting odemis with no GUI
+----------------------------------------
+
+The odemis backend without launching the GUI can be started with the following
```

Comment:
I'd suggest to reorder the sentence as `The odemis backend can be started without launching the GUI, by using the following::`
Note that it doesn't need to be called from `~/development/odemis`. The previous command needs it because of the path to the YAML file. Maybe it'd be clearer to just also write the previous command as:
odemis-start ~/development/odemis/install/linux/usr/share/odemis/sim/sparc2-sim.odm.yaml